### PR TITLE
[Post v1.1.0] Run the sample YAML tests with KubeRay v1.1.0

### PR DESCRIPTION
**Pull request by kevin85421**
## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/Makefile [hunk 1]
```diff
@@ -156,7 +156,7 @@ kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
 ENVTEST = $(LOCALBIN)/setup-envtest
 $(ENVTEST): $(LOCALBIN)
 envtest: $(ENVTEST) ## Download envtest locally if necessary.
-	test -s $(ENVTEST) || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+	test -s $(ENVTEST) || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20240201105228-4000e996a202
```
Comment (jjyao) [inline ray-operator/Makefile:159]: Why this change?

**Review by jjyao — APPROVED** [review]

File: ray-operator/Makefile (hunk 1 above)
Comment (kevin85421) [reply ray-operator/Makefile:159]: See https://github.com/ray-project/kuberay/pull/2038 for more details.
